stl/inc/chrono

```diff
@@ -3472,66 +3472,55 @@ namespace chrono {
             const _LeapSecondRep _Original_leap{_Leap};
             if (_Leap == _LeapSecondRep::_File_time) {
                 const int _Year{*_Century + *_Two_dig_year};
-                _Leap = _Year <= 2016 ? _LeapSecondRep::_Negative : _LeapSecondRep::_All;
+                _Leap = _Year < 2018 ? _LeapSecondRep::_None : _LeapSecondRep::_All;

```

Comment:
This doesn't implement the thing written in the comment above; this is `_None` before 2018-01-01, `_All` afterwards (though there isn't a change in behavior, obviously)